grammar.js

```diff
@@ -1094,6 +1115,94 @@ module.exports = grammar({
       ),
     )),
 
+    _with_settings: $ => seq(
+          field('name', $.identifier),
+          optional('='),
+          field('value', choice($.identifier, alias($._single_quote_string, $.literal))),
+    ),
+
+    create_database: $ => seq(
+      $.keyword_create,
+      $.keyword_database,
+      $.identifier,
+      optional($.keyword_with),
+      repeat(
+        $._with_settings
+      ),
```

Comment:
```suggestion
      optional(
        seq(
          $.keyword_with,
          repeat1(
            $._with_settings
          ),
        ),
      ),
```
prevent settings without `with` -- I think this goes for some of the other additions too?

Reply:
I think this is wrong: 
https://www.postgresql.org/docs/current/sql-createdatabase.html

I think the `with` is optional and then there can be 0 or more `_with_options`
```
CREATE DATABASE name
    [ WITH ] [ OWNER [=] user_name ]
           [ TEMPLATE [=] template ]
...
```